mino/minogrpc/overlay.go

```diff
@@ -89,35 +102,59 @@ func (o overlayService) Stream(stream Overlay_StreamServer) error {
 			"header. Expected 1, found %d", len(apiURI))
 	}
 
+	rcvr, ok := o.handlerRcvr[apiURI[0]]
+	if ok {
+		go func() {
```

Comment:
This logic should be separated from the receiver. A receiver transmits the message addressed to the node, it announces the end of the stream, etc .. Everything that a client should know.

This logic is to relay a message to a different node which is a totally different meaning.